src/app-layout/mobile-toolbar/index.tsx

```diff
@@ -106,6 +110,25 @@ export function MobileToolbar({
           onClick={onToolsOpen}
         />
       )}
+      {drawers && (
+        <aside aria-label="Drawers" className={clsx(styles['mobile-toggle'], styles['mobile-toggle-with-drawers'])}>
```

Comment:
1. Another hard-coded label 
2. Can we avoid this wrapper all together and have a flat layout?

```html
<div className={clsx(styles['mobile-bar']}>
   <MobileToggle/> //navigation
   <div className={styles['mobile-bar-breadcrumbs']}></div>
   {drawers.map(toggleConfig = > <MobileToggle {...toggleConfig} />}
</div>
```

The `drawers` array need to be normalised to include tools in a single place, according to the comment above (TODO link)

Reply:
Makes sense.

Reply:
Note: I updated the hard-coded label, but keeping the structure as before. To change in a follow-up.